mne_bids/tests/test_read.py

```diff
@@ -295,19 +295,60 @@ def test_handle_coords_reading():
     write_raw_bids(raw, bids_basename, bids_root, overwrite=True)
 
     # read in the data and assert montage is the same
+    # regardless of 'm', 'cm', 'mm', or 'pixel'
     bids_fname = bids_basename + "_ieeg.edf"
-    raw_test = read_raw_bids(bids_fname, bids_root)
-
-    # obtain the sensor positions
-    orig_locs = raw.info['dig'][1]
-    test_locs = raw_test.info['dig'][1]
-    assert orig_locs == test_locs
-    assert not object_diff(raw.info['chs'], raw_test.info['chs'])
-
-    # test error message if electrodes don't match
+    coordinate_units = ['m', 'cm', 'mm']
+    scaling_numbers = [1., 100., 1000.]
+    coordsystem_fname = _find_matching_sidecar(bids_fname, bids_root,
+                                               suffix='coordsystem.json')
     electrodes_fname = _find_matching_sidecar(bids_fname, bids_root,
                                               "electrodes.tsv",
                                               allow_fail=True)
+    orig_electrodes_dict = _from_tsv(electrodes_fname,
+                                     [str, float, float, float, str])
+    for coord_unit, scaling in zip(coordinate_units, scaling_numbers):
+        # update coordinate units
+        _update_sidecar(coordsystem_fname, 'iEEGCoordinateUnits', coord_unit)
+        electrodes_dict = _from_tsv(electrodes_fname,
+                                    [str, float, float, float, str])
+        for axis in ['x', 'y', 'z']:
+            electrodes_dict[axis] = np.multiply(orig_electrodes_dict[axis],
+                                                scaling)
+        _to_tsv(electrodes_dict, electrodes_fname)
+
+        # read in raw file w/ updated montage
+        raw_test = read_raw_bids(bids_fname, bids_root)
+
+        # obtain the sensor positions and make sure they're the same
+        for origdig, newdig in zip(raw.info['dig'], raw_test.info['dig']):
+            np.testing.assert_array_equal(origdig, newdig)
+
+    # check that coordinate systems can be used and defaults to mri
+    coordinate_frames = ['lia', 'ria', 'lip', 'rip', 'las']
```

Comment:
can you add a link to the documentation of these coordinate systems (from BIDS specification?) in the comments where you define them?